### ISSUE TITLE
Open Graph Protocole metadata is missing

### ISSUE THREAD

**Issue by jorgejesus** [ISRICWorldSoil/SoilGrids250m#17]
The ISRIC websites require the webstats content and ogp content to be set on the index.html
See http://ogp.me/ for extra information

Normally the Facebook validator can help to determine if content is ok
https://developers.facebook.com/tools/debug/

For debugging I have to remove the password from the site.

Example of OGP + Twitter cards:
```
<meta charset="utf-8">
<meta content="IE=edge,chrome=1" http-equiv="X-UA-Compatible">
<meta content="width=device-width, initial-scale=1.0" name="viewport">
<meta content="World Soil Museum, generic museum information and content online for visual tour" name="description">
<meta content="ISRIC,World Soil Museum,Soil,Museum,Monoliths" name="keywords">
<meta content="Jorge Samuel Mendes de Jesus, Ingrid Haas" name="author">
<meta content="World Soil Museum | ISRIC" property="og:title">
<meta content="website" property="og:type">
<meta content="1008952465863297" property="fb:app_id">
<meta content="http://wsm.isric.org/images/misc/World_Soil_Museum_ISRIC.png" property="og:image">
<meta content="http://wsm.isric.org/" property="og:url">
<meta content="" property="og:description">
<meta content="summary" name="twitter:card">
<meta content="@WorldSoilMuseum" name="twitter:site">
<meta content="World Soil Museum Website" name="twitter:title">
<meta content="World Soil Museum, generic museum information and content online for visual tour" name="twitter:description">
<meta content="http://wsm.isric.org/images/misc/World_Soil_Museum_ISRIC.png" name="twitter:image">
```
@thengl  You could provide the screenshoot that you want the social media to display

**Comment by aL3xa** [comment]
Fixed in 3e501bb

**Comment by thengl** [comment]
Jorge forgot to mention that the metadata needs to be adopted to:

```
<meta charset="utf-8">
<meta content="IE=edge,chrome=1" http-equiv="X-UA-Compatible">
<meta content="width=device-width, initial-scale=1.0" name="viewport">
<meta content="SoilGrids: global gridded soil information" name="description">
<meta content="ISRIC,SoilGrids,Data,Maps" name="keywords">
<meta content="Tomislav Hengl, Jorge Samuel Mendes de Jesus" name="author">
<meta content="SoilGrids | ISRIC" property="og:title">
<meta content="website" property="og:type">
<meta content="http://www.isric.org/sites/default/files/SoilGrids_webmapping.png" property="og:image">
<meta content="http://soilgrids.org" property="og:url">
<meta content="" property="og:description">
<meta content="summary" name="twitter:card">
<meta content="@ISRICorg" name="twitter:site">
<meta content="SoilGrids" name="twitter:title">
<meta content="Global gridded soil information" name="twitter:description">
<meta content="http://www.isric.org/sites/default/files/SoilGrids_webmapping.png" name="twitter:image">
```

**Comment by aL3xa** [comment]
Fixed in 0d11a35
